architecture/additional_concepts/authorization.adoc

```diff
@@ -295,158 +469,158 @@ The verbs are described in the following table.
+replicationcontrollers/scale statefulsets rolebindings roles buildconfigs
+buildconfigs/webhooks builds deploymentconfigs deploymentconfigs/scale
+generatedeploymentconfigs imagestreamimages imagestreammappings imagestreams
+imagestreams/secrets imagestreamtags imagestreams/layers projects routes
+routes/status processedtemplates templateconfigs templates buildlogs
+
 |watch
-|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints 
-persistentvolumeclaims replicationcontrollers replicationcontrollers/scale 
-secrets serviceaccounts services services/proxy bindings events limitranges 
-namespaces namespaces/status pods/log pods/status replicationcontrollers/status 
-resourcequotas resourcequotas/status horizontalpodautoscalers cronjobs jobs 
-scheduledjobs deployments deployments/rollback deployments/scale 
-horizontalpodautoscalers jobs replicasets replicasets/scale 
-replicationcontrollers/scale daemonsets statefulsets rolebindings roles 
-policies policybindings rolebindingrestrictions buildconfigs 
-buildconfigs/webhooks builds builds/log deploymentconfigs 
-deploymentconfigs/scale generatedeploymentconfigs deploymentconfigs/log 
-deploymentconfigs/status imagestreamimages imagestreammappings imagestreams 
-imagestreams/secrets imagestreamtags imagestreams/status 
-appliedclusterresourcequotas routes routes/status processedtemplates 
-templateconfigs templates buildlogs resourcequotausages 
- 
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints
+persistentvolumeclaims replicationcontrollers replicationcontrollers/scale
+secrets serviceaccounts services services/proxy bindings events limitranges
+namespaces namespaces/status pods/log pods/status replicationcontrollers/status
+resourcequotas resourcequotas/status horizontalpodautoscalers cronjobs jobs
+scheduledjobs deployments deployments/rollback deployments/scale
+horizontalpodautoscalers jobs replicasets replicasets/scale
+replicationcontrollers/scale daemonsets statefulsets rolebindings roles
+policies policybindings rolebindingrestrictions buildconfigs
+buildconfigs/webhooks builds builds/log deploymentconfigs
+deploymentconfigs/scale generatedeploymentconfigs deploymentconfigs/log
+deploymentconfigs/status imagestreamimages imagestreammappings imagestreams
+imagestreams/secrets imagestreamtags imagestreams/status
+appliedclusterresourcequotas routes routes/status processedtemplates
+templateconfigs templates buildlogs resourcequotausages
+
 |===
 
 ifdef::openshift-enterprise,openshift-origin[]
 .basic-user Rules Matrix
 [cols="2s,8l"]
 |===
 |create
-|selfsubjectrulesreviews localsubjectaccessreviews subjectaccessreviews 
- 
+|selfsubjectrulesreviews localsubjectaccessreviews subjectaccessreviews
+
 |get
-|users clusterroles 
- 
+|users clusterroles
+
 |list
-|projectrequests clusterroles storageclasses projects 
- 
+|projectrequests clusterroles storageclasses projects
+
 |watch
-|projects 
- 
+|projects
+
 |===
 
 endif::[]
```

Comment:
The cluster-admin Rules Matrix is rendering as a blank table in the preview.

Reply:
Interesting. Thanks @bmcelvee I will double-check the formatting there.